### PR TITLE
Validating fingerprint before adding an item to agent implemented. AEA-536

### PR DESCRIPTION
**Pull request by panasevychol**
## Proposed changes

Validating fingerprint before adding an item to agent implemented.
#1069 

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: aea/cli/add.py [hunk 1]
```diff
@@ -131,26 +152,25 @@ def _add_item(click_context, item_type, item_public_id) -> None:
     # find and add protocol
     if item_public_id in [DEFAULT_CONNECTION, DEFAULT_PROTOCOL, DEFAULT_SKILL]:
         package_path = _find_item_in_distribution(ctx, item_type, item_public_id)
-        _copy_package_directory(
+        package_path = _copy_package_directory(
```
Comment (DavidMinarsch) [inline aea/cli/add.py:155]: revert this line

File: aea/cli/add.py [hunk 2]
```diff
@@ -131,26 +152,25 @@ def _add_item(click_context, item_type, item_public_id) -> None:
...
             ctx, package_path, item_type, item_public_id.name, item_public_id.author
         )
     elif is_local:
         package_path = _find_item_locally(ctx, item_type, item_public_id)
-        _copy_package_directory(
+        package_path = _copy_package_directory(
```
Comment (DavidMinarsch) [inline aea/cli/add.py:160]: revert this line

File: aea/cli/add.py [hunk 3]
```diff
@@ -131,26 +152,25 @@ def _add_item(click_context, item_type, item_public_id) -> None:
     # find and add protocol
     if item_public_id in [DEFAULT_CONNECTION, DEFAULT_PROTOCOL, DEFAULT_SKILL]:
         package_path = _find_item_in_distribution(ctx, item_type, item_public_id)
```
Comment (DavidMinarsch) [inline aea/cli/add.py:154]: call this source_path

File: aea/cli/add.py [hunk 4]
```diff
@@ -131,26 +152,25 @@ def _add_item(click_context, item_type, item_public_id) -> None:
...
-        _copy_package_directory(
+        package_path = _copy_package_directory(
             ctx, package_path, item_type, item_public_id.name, item_public_id.author
         )
     elif is_local:
         package_path = _find_item_locally(ctx, item_type, item_public_id)
```
Comment (DavidMinarsch) [inline aea/cli/add.py:159]: call this source path

File: aea/cli/common.py [hunk 5]
```diff
@@ -498,6 +498,7 @@ def _copy_package_directory(ctx, package_path, item_type, item_name, author_name
         sys.exit(1)
 
     Path(ctx.cwd, "vendor", author_name, item_type_plural, "__init__.py").touch()
+    return Path(dest)
```
Comment (DavidMinarsch) [inline aea/cli/common.py:501]: Function signature return value and docstring has not been updated

Comment (panasevychol) [reply aea/cli/common.py:501]: Dockstring is right there.
https://github.com/fetchai/agents-aea/pull/1131/files/75a03097a0be8b9ecc3a32ddbd2e888f49cad788#diff-7569427e56f7d9e415d5c8f482d1153cR486
Function signature - wasn't there before. Will add.

**Pull request closed without merging**